xchange-ftx/src/main/java/org/knowm/xchange/ftx/service/FtxDigest.java

```diff
@@ -30,7 +30,7 @@ public String digestParams(RestInvocation restInvocation) {
             + restInvocation.getHttpMethod().toUpperCase()
             + restInvocation.getPath();
 
-    if(restInvocation.getQueryString() != null){
+    if(!restInvocation.getQueryString().isEmpty()){
```

Comment:
can you check if ! null in front as well

Reply:
it is never null